src/main/java/mate/academy/spring/controller/UserController.java

```diff
@@ -0,0 +1,62 @@
+package mate.academy.spring.controller;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.model.User;
+import mate.academy.spring.model.dto.UserResponseDto;
+import mate.academy.spring.service.UserService;
+import mate.academy.spring.service.mapper.UserDtoMapper;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/users")
+public class UserController {
+    private final UserService userService;
+    private final UserDtoMapper userDtoMapper;
+
+    public UserController(UserService userService,
+                          UserDtoMapper userDtoMapper) {
+        this.userService = userService;
+        this.userDtoMapper = userDtoMapper;
+    }
+
+    @GetMapping("/")
+    public List<UserResponseDto> getAllUser() {
+        return userService.listUsers().stream()
+                .map(userDtoMapper::parse)
+                .collect(Collectors.toList());
+    }
+
+    @GetMapping("/{id}")
+    public UserResponseDto get(@PathVariable Long id) {
+        return userDtoMapper.parse(userService.get(id).get());
```

Comment:
```suggestion
        return userDtoMapper.parseToDto(userService.get(id).get());
```
we'll have response and request DTOs, so it's good to specify which way you map the object

Reply:
Done